compiler/crates/relay-typegen/tests/generate_typescript/fixtures/default-input.graphql

```diff
@@ -0,0 +1,7 @@
+mutation feedbackUnLikeMutation($input: FeedbackUnLikeInput) {
+  feedbackUnLike(input: $input) {
+    feedback {
+      id
+    }
+  }
+}
```

Comment:
Would you mind adding this same fixture to the Flow tests?

https://github.com/facebook/relay/tree/main/compiler/crates/relay-typegen/tests/generate_flow/fixtures

Reply:
arg, my bad... here they are :)